docs: update v9 styling guide to include griffel devtools extension link (#23878)
microsoft/fluentui: apps/public-docsite-v9/src/Concepts/StylingComponents.stories.mdx

}
```

### Debugging styles

> [Griffel devtools chrome extension](https://chrome.google.com/webstore/detail/griffel-devtools/bejhagjehnpgagkaaeehdpdadmffbigb) is especially useful for debugging style override. It shows all griffel styles applied on the currently selected DOM element, including the styles that are overridden in `mergeClasses`.

Reviewer: ```suggestion
[Griffel devtools chrome extension](https://chrome.google.com/webstore/detail/griffel-devtools/bejhagjehnpgagkaaeehdpdadmffbigb) can be used to debug style overrides. It shows all griffel styles applied on the currently selected DOM element, including the styles that are overridden in `mergeClasses`.
```

Author: Somehow this comment wasn't refreshed when I hit merged :/ I opened a new PR to re-word it as suggested: #23887
